.github/workflows/create-release.yml

```diff
@@ -1,96 +1,78 @@
-name: "Create release"
+name: Create release
 
 on:
   workflow_dispatch:
+    inputs:
+      DRY_RUN:
+        type: boolean
+        default: false
+        description: "Dry run: setting this to `true` will skip the final step of publishing the release, leaving it as a draft-release. This can be helpful for testing and debugging. Note that the tag and release are still getting generated and the next run of this result in incremented release version. To avoid this, do not forget to delete the resulting release and tag."
 
 jobs:
-  verify-release:
-    name: Verify release
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.gen-version.outputs.VERSION }}
+  gen-version:
+    name: Generate semantic version from branch and tags
+    uses: kyma-project/eventing-tools/.github/workflows/get-version-from-release-branch-reusable.yml@main
+    # outputs: VERSION
 
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+  bump-sec-scanners-config:
+    name: Bump the sec-scandners-config.yaml
+    needs: gen-version
+    uses: kyma-project/eventing-tools/.github/workflows/bump-sec-scanners-config-reusable.yml@main
+    with:
+      VERSION: ${{ needs.gen-version.outputs.VERSION }}
+    secrets:
+      BOT_PAT: ${{ secrets.BOT_PAT }}
 
-      - name: Generate version number
-        id: gen-version
-        run: |
-          # get script
-          GET_VERSION=$(mktemp /tmp/get-version-from-branch.XXXXX)
-          curl -L https://raw.githubusercontent.com/kyma-project/eventing-tools/main/hack/scripts/get-version-from-branch.sh -o "${GET_VERSION}"
-          chmod +x "${GET_VERSION}"
-          # get version via script
-          VERSION=$("${GET_VERSION}")
-          # push version to output environment file
-          echo "VERSION=${VERSION}" >> $GITHUB_OUTPUT
+  run-unit-test:
+    name: Run Unit Tests
+    needs: [gen-version, bump-sec-scanners-config]
```

Comment:
Why does `run-unit-test` need `bump-sec-scanners-config`?

Reply:
It does not **need**, it will just wait to run the unit tests after we do the PR in the `bump-sec-scanners-config`. In general, all the jobs will run consecutive. Only `lint` and `unit-test` will run in parallel because they are independent of each other. That flow is controlled by the `need` instructions.

Reply:
the only flow that is really **need**ed is the `gen-version` because only if it get's marked as `needs` the output (`VERSION`) gets available in the other jobs.